main.go

```diff
@@ -54,7 +55,7 @@ func getKubectl() (*kubernetes.Clientset, error) {
 
 func main() {
 	var (
-		provider          string
+		_provider         string
```

Comment:
why does this have an _ as a prefix?

Reply:
@sorenmat 
provider variable collides with imported package `"github.com/sorenmat/k8s-rds/provider"`, it is better to change the variable name to avoid collision.